docs/26_export_manual.md

```diff
@@ -0,0 +1,296 @@
+% Data Export
+% Colorado Risk Limiting Audit Tool 
+% 2017
+
+Separate from the RLA application server and client software that supports the Department of State and the Counties in carrying out the Risk Limiting Audit, there is a command, called `rla_export`, allowing export of data from the central server and the underlying database.
+
+The command is part of a python package, whose technical description can be found in a `README.rst` file in the python site-packages directory tree wherever the package is installed. The most current online version (which may or may not match the version you have installed) is available 
+in the [public code repository](https://github.com/FreeAndFair/ColoradoRLA/tree/auditcenter/server/eclipse-project/script/rla_export).
+
+The `README.rst` file gives instructions for installing the python package, and describes various run-time options. For a catalog of the exports produced by the command, see below.
+
+
+# Minimum Data Required to Allow Public to Reproduce Audit Calculations
+
+To allow independent verification of the RLA, 
+the Colorado Department of State must provide to the following to the public :
+
+- all CVR files and their hashes
+- all ballot manifest files and their hashes
+- the list of contests selected for audit, and which if any have been designated for hand count
+- opportunity to observe the random seed selection
+- for each County, the random sequence of ballot cards determined by the random seed and the pseudo-random number generator
+- opportunity to observe the activities of the County Audit Boards
+- announced tabulated results for contests selected for audit
+- the risk limit
+- the error inflation factor
+- the tabulation of results and counts of ballot cards used to calculate the diluted margin (including the number of winners for each contest selected for audit)
+- for each County and each round of the audit, the list of ballot cards assigned to the Audit Board for review
+- for each contest selected for audit, and for each cast vote record that contains the given contest and has been presented to the Audit Board for verification, the RLA system's record of the Audit Board's review of the physical ballot for that contest
+
+
+# Exports
+The ``rla_export`` command exports many of the files necessary for independent verification of 
+the RLA, whether by candidates, parties, other organizations. 
+
+## Database Exports in.csv and .json format
+
+These data files in this section are generated based on `sql` query files.
+These are always produced in two formats: ``.json` and ``.csv``.
```

Comment:
there are some double accents here and on the next few lines instead of the single accents that markdown uses.